docs/app_metrics_guide.adoc

```diff
@@ -0,0 +1,41 @@
+= Understanding & taking actions based on App Metrics
+
+This guide gives details on a number scenarios where App Metrics can be analysed to see if actions can or should be taken by the App developer.
+
+[discrete]
+== Prerequisites
+
+* A running Metrics setup i.e. a provision of the Metrics APB
+* An App that is using the Metrics SDK hooked up to the App Metrics server
```

Comment:
is 'metrics sdk' a term we want to use for client sdks?

Reply:
To date, I've heard & used the following SDK terms:
* Core SDK
* Auth SDK
* Metrics SDK
* Android SDK
* Android Core SDK
* Android Metrics SDK
* Android Auth SDK
* iOS SDK
* iOS Core SDK
* iOS Auth SDK
* iOS Metrics SDK
* JS SDK
* Cordova SDK
* React Native SDK
* Web SDK
* Aerogear SDK
* Aerogear Services SDK
* AeroGear Services Core
* Aerogear Services Sync

plus many other variations.
There's a mail thread that discusses some of this https://groups.google.com/forum/#!msg/aerogear/pNmVTy0saDA/pIU92HXOAgAJ;context-place=forum/aerogear

What I want to convey in this prerequisite is the idea that the developers App (doesn't mater the platform) has the necessary bits in it to send Metrics to the App Metrics service

Reply:
@david-martin seems like two things:
1. we don't know what an sdk is (or at least how to name one)
2. we need to explicitly tell users what bits to put into their app to make metrics work. 

re. 1 - big discussion that we'll deal with outside of this PR
re. 2 - do we need a  'How to configure your app to use your metrics service' topic ?  would this be something generic? or specific to each client sdk?

Reply:
> re. 2 - do we need a 'How to configure your app to use your metrics service' topic ? would this be something generic? or specific to each client sdk?

Yes, we do. It would be specific to each client sdk.
For iOS, there is this so far https://github.com/aerogear/aerogear-ios-sdk/pull/21/files I'm not sure that captures "How to configure your app to use your metrics service", so it may need restructuring (based on one a template)?

For Android, it looks like a similar story
https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/getting-started/metrics.adoc
It feels like these need to be rewritten/restructured to answer the qeustion of 'How to configure your app to use your metrics service'

Thoughts?